store/sql_store.go

```diff
@@ -127,6 +123,11 @@ func setupConnection(con_type string, driver string, dataSource string, maxIdle
 }
 
 func (ss SqlStore) DoesColumnExist(tableName string, columnName string) bool {
+	// XXX TODO FIXME this should be removed after 0.6.0
```

Comment:
This refers to the entire function and will be removed when we implement database schema versions. Correct?